src/natcap/invest/globio.py

```diff
@@ -939,22 +939,29 @@ def _collapse_infrastructure_layers(
 
     def _collapse_infrastructure_op(*infrastructure_array_list):
         """For each pixel, create mask 1 if all valid, else set to nodata."""
-        nodata_mask = (
-            numpy.isclose(
-                infrastructure_array_list[0], infrastructure_nodata_list[0]))
-        infrastructure_result = infrastructure_array_list[0] > 0
```

Comment:
Hi, this comment is out of order, but it's the last one I'm doing after I parsed through the whole thing. If I understand the issue, would a single change of this line to `infrastructure_result = (infrastructure_array_list[0] > 0) & ~nodata_mask` fix the issue?

Reply:
Dang, that's a great paren catch on a bug that I didn't realize was actually there. I actually just posted about this in slack before reading over your comments again.

Reply:
This is patched here with proper parens: https://github.com/natcap/invest/pull/186/commits/86dab80e89b22c452e5da5b2efec78decbfa62cb